pyscriptjs/tests/integration/test_py_repl.py

```diff
@@ -8,61 +8,78 @@ class TestPyRepl(PyScriptTest):
     def test_repl_loads(self):
```

Comment:
This entire file looks good to me 👍 
Should I review the rest too?

Reply:
> This entire file looks good to me
🎉

> Should I review the rest too?
reviews are always welcome :)